flux_led/utils.py

```diff
@@ -200,14 +261,7 @@ def rgbww_brightness(
             (255 - cw_brightness) * change_brightness_pct + cw_brightness
         )
 
-    r, g, b = colorsys.hsv_to_rgb(h, s, color_brightness / 255)
-    return (
-        round(r * 255),
-        round(g * 255),
-        round(b * 255),
-        ww_brightness,
-        cw_brightness,
-    )
+    return color_brightness, cw_brightness, ww_brightness
 
 
 def color_temp_to_white_levels(temperature, brightness):
```

Comment:
So it looks like like this was reversed but so was https://github.com/Danielhiversen/flux_led/pull/117/files#diff-7545dd8de606048638bdd92c9289888e183c836a324b7395873c14de4aaa2f97R460 so it worked anyways

Reply:
```
bdraco@travelmacjnk-5 flux_led % flux_led 192.168.106.210 --CCT 6500 255 -i
Setting LED temperature 6500K and brightness: 255%
Unknown ID [192.168.106.210] ON  [Color: (0, 0, 0) White: 0 raw state: 129,37,35,97,5,16,0,0,0,0,2,255,15,79,]
bdraco@travelmacjnk-5 flux_led % flux_led 192.168.106.210 --CCT 2700 255 -i
Setting LED temperature 2700K and brightness: 255%
Unknown ID [192.168.106.210] ON  [Color: (0, 0, 0) White: 255 raw state: 129,37,35,97,5,16,0,0,0,255,2,0,15,79,]
```